pynecone/state.py

```diff
@@ -368,13 +368,17 @@ def get_delta(self) -> Delta:
         Returns:
             The delta for the state.
         """
+        delta = {}
+
         # Return the dirty vars, as well as all computed vars.
-        delta = {
-            self.get_full_name(): {
-                prop: getattr(self, prop)
-                for prop in self.dirty_vars | set(self.computed_vars.keys())
+        if len(self.dirty_vars) > 0:
```

Comment:
I just realized this may break the clock app - maybe I'll get rid of this `if` statement.

Reply:
I removed this to always include computed vars.